docker/gremlin-server/gremlin-server-integration.yaml

```diff
@@ -24,8 +24,7 @@ graphs: {
   modern: conf/tinkergraph-empty.properties,
   crew: conf/tinkergraph-empty.properties,
   grateful: conf/tinkergraph-empty.properties,
-  sink: conf/tinkergraph-empty.properties
-}
+  sink: conf/tinkergraph-empty.properties}
```

Comment:
Was this autoformat?

Reply:
no, I tried to change something and then made like in other configs: https://github.com/lyndonbauto/tinkerpop/blob/master/docker/gremlin-server/gremlin-server-integration-secure.yaml#L27

Reply:
reverted to reduce noise